stackslib/src/clarity_vm/clarity.rs

```diff
@@ -294,6 +295,26 @@ impl ClarityInstance {
             )
         };
 
+        // If we're starting a new tenure, and we're in epoch 3.09, then we
```

Comment:
Typo -- epoch 3.09?

Reply:
Fixed in 7a63f899e8945a770642517e96324330597dbe87.